bin/oneoff/census/state-data/reseed-state-data.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../../dashboard/config/environment'
```

Comment:
at this point maybe absolute path is better?

Reply:
Is there an absolute path that will work in all environments?

Reply:
After a bunch of searching, apparently not.  Pegasus implemented a root `src_dir` but I don't see one for other environments.  Not to worry then.